src/App.js

```diff
@@ -14,11 +15,58 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    choosenItem: '',
+  };
+
+  chooseItem = (event) => {
+    this.setState({ choosenItem: event.target.innerText });
+  };
+
+  clearSelection = (event) => {
+    this.setState({ choosenItem: '' });
+  };
+
+  render() {
+    const { choosenItem } = this.state;
+
+    return (
+      <div className="App">
+        <div className="App__header header">
+          <h1>
+            Selected good:
+            {' '}
+            {choosenItem || '-'}
+          </h1>
```

Comment:
Якщо ти початково в state.choosenItem запишеш '-', а не просто пустий рядок, то тобі не треба буде робити цієї перевірки. Тоді можна легко це в один рядок буде записати
```suggestion
          <h1>
            {`Selected good: ${choosenItem}`}
          </h1>
```

Reply:
тоді мені прийдеться в 3ьох місцях писати цей '-', що не зовсім зручно, якщо захочеться поміняти цей знак